src/core/login/pages/reconnect/reconnect.ts

```diff
@@ -265,7 +265,7 @@ export class CoreLoginReconnectPage {
     async validateCredentialsOffline(username: string, password: string): Promise<boolean> {
         try {
             const storedHashedCredentials = await this.offlineAuthProvider.getHashedCredentials(this.siteId);
-            const enteredHashedCredentials = this.hashCredentials(username, password);
```

Comment:
are you only storing the password hash now? why the change?

Reply:
did this to simplify things. in the password reset flows, there is no username on the redhat sso side. so it can't be used as a salt. If we need to, we can add the salt on the moodleapp side. the easiest would be to use the siteId.